src/org/usfirst/frc/team1923/robot/commands/drive/DriveTimeCommand.java

```diff
@@ -24,33 +27,51 @@ public DriveTimeCommand(double power, double timeOut) {
         this.power = power;
     }
 
+    /**
+     * Drives a set time at a set speed
+     *
+     * @param power
+     *            PercentVBus value for both motors
+     * @param timeOut
+     *            Timeout in seconds
+     * @param isLow
+     *            set the gears to low or high
+     */
     public DriveTimeCommand(double power, double timeOut, boolean isLow) {
 
         requires(Robot.driveSubSys);
-        setTimeout(timeOut);
+        setTimeout(timeOut); // set time to drive

```

Comment:
This kind of commenting really doesn't do anything because the code is self-explanatory. Please limit your comments to non-obvious aspects of the code aka structure/decision process